streaming-binlog/src/main/java/org/openmrs/analytics/GetUuidUtil.java

```diff
@@ -0,0 +1,51 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package org.openmrs.analytics;
+
+import java.beans.PropertyVetoException;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.Map;
+
+public class GetUuidUtil {
+	
+	private JdbcConnectionUtil jdbcConnectionUtil;
+	
+	public GetUuidUtil(JdbcConnectionUtil jdbcConnectionUtil) {
+		this.jdbcConnectionUtil = jdbcConnectionUtil;
+	}
+	
+	public String getUuid(String parentTable, String parentForeignKey, String childPrimaryKey, Map payload)
+	        throws PropertyVetoException, SQLException, ClassNotFoundException {
+		
+		Statement stmt = jdbcConnectionUtil.createStatement();
+		String uuidResultFromSql = null;
+		
+		String sql = String.format("SELECT uuid FROM %s WHERE %s = %s", parentTable, parentForeignKey,
+		    payload.get(childPrimaryKey.toString()));
+		
+		ResultSet rs = stmt.executeQuery(sql);
+		while (rs.next()) {
+			uuidResultFromSql = rs.getString("uuid");
+		}
+		rs.close();
+		stmt.close();
+		
+		return uuidResultFromSql;
```

Comment:
Please either throw an exception if `uuidResultFromSql` is `null` here (preferred) or account for a `null` value being returned by this method in `FhirConverter` (not-preferred unless it is really a legitimate case).